change/@fluentui-react-motion-components-preview-27662e3f-3d93-496c-8b66-b5de8fac23de.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33133&commitId=3d15d6f288edad8d1a8e3a785445af6cc598a2a9&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Basic High Contrast.chromium.png|2| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|4| Changed|
|Avatar Converged.badgeMask.chromium.png|2| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|975| Changed|
|Drawer.overlay drawer full.chromium.png|7| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>